codegen/src/main/java/software/amazon/awssdk/codegen/poet/rules/EndpointRulesSpecUtils.java

```diff
@@ -410,4 +423,19 @@ private CodeBlock parameterDefaultValueCode(ParameterModel parameterModel) {
     public String variableName(String name) {
         return intermediateModel.getNamingStrategy().getVariableName(name);
     }
+
+    private void handleArrayDefaultValue(CodeBlock.Builder b, String parameterType, JrsArray defaultValue) {
+        if ("stringarray".equalsIgnoreCase(parameterType)) {
+            Iterator<JrsValue> elementValuesIter = defaultValue.elements();
+            b.add("$T.asList(", Arrays.class);
+            while (elementValuesIter.hasNext()) {
+                JrsValue v = elementValuesIter.next();
+                b.add("\"" + v.asText() + "\"");
+                if (elementValuesIter.hasNext()) {
+                    b.add(",");
```

Comment:
nit, seems like we can use stringjoiner?

Reply:
Done